### PR TITLE
adding 'conda-forge,cdat_dev' for channel_sources

### PR DESCRIPTION
**Pull request by muryanto1**
adding 'conda-forge,cdat_dev' for channel_sources, because we need the build to pick up cdat_info from conda-forge/label/cdat_dev

Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by muryanto1** [comment]
@conda-forge-admin, please rerender

**Comment by muryanto1** [comment]
@conda-forge-admin, please rerender

**Comment by muryanto1** [comment]
@isuruf  if I look at the build log, it is picking up the following cdat_info:
    cdat_info:          8.2-py_7                  conda-forge

but I want the build to pick up cdat_info that is from conda-forge/label/cdat_dev and I did specify that in recipe/conda_build_config.yaml for the channel_sources.
Let me know what I need to do so that it picks up cdat_info from conda-forge/label/cdat_dev.
Thanks.

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -1,2 +1,8 @@
 channel_targets:
   - conda-forge cdat_dev
+channel_sources:
+  - conda-forge/label/cdat_dev
+  - conda-forge,defaults                        # [not (aarch64 or armv7l)]
```
Comment (isuruf) [inline recipe/conda_build_config.yaml]:
```suggestion
  - conda-forge/label/cdat_dev,conda-forge,defaults                        # [not (aarch64 or armv7l)]
```

**Comment by muryanto1** [comment]
@conda-forge-admin, please rerender

File: recipe/conda_build_config.yaml (hunk 1 above)
Comment (muryanto1) [reply recipe/conda_build_config.yaml]: @isuruf cool, that fixed it. thanks!

**Comment by muryanto1** [comment]
@isuruf where can I check the status of the build that will get build 1 uploaded to conda-forge/label/cdat_dev?
it has been 30 minutes, and I have not seen the packages get uploaded yet. Thanks.
I see build logs here:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=175783&view=results
but build 1 did not get uploaded to conda-forge/label/cdat_dev

**Comment by xylar** [comment]
@muryanto1, you would look at the list of commits.  You can see a green check mark after the merge commit for this PR:
https://github.com/conda-forge/cdtime-feedstock/commits/dev
That means CI has finished successfully.  If you mouse over the green check, you should be able to bring up details and determine that the upload happened.

**Comment by muryanto1** [comment]
@xylar  build 1 did not get uploaded -- I am not seeing it here https://anaconda.org/conda-forge/cdtime/files  
I am seeing this line in the log
+ upload_package --validate --feedstock-name=cdtime-feedstock /home/conda/feedstock_root /home/conda/recipe_root /home/conda/feedstock_root/.ci_support/linux_numpy1.14python3.6.____cpython.yaml
Uploading to conda-forge with source channel 'default' is not allowed

**Comment by isuruf** [comment]
`default` -> `defaults`

**Comment by muryanto1** [comment]
> `default` -> `defaults`

@isuruf aargh..thanks!

**Pull request merged**